### ISSUE TITLE
Feature request: add possibility to access favorites via a command

### ISSUE THREAD

**Issue by carlfriedrich** [leftstick/vscode-favorites#59]
I really like the simplicity of this extension. Adding and removing via the according commands works fine. However, I am not able to access the favorites via a command. As far as I can see, the only way of accessing the favorites is through the UI widget.

Would it be possible to add a command, e.g. "Open Favorites", which displays a list of all the favorites from which I can choose a file to open then? This way it would also be possible to add a keybinding for it, which would greatly improve efficiency.

**Comment by leftstick** [comment]
@carlfriedrich  thanks for your question. I think the concern is that folder is allowed to added in favorites list. 
If you use command, only favorite files can be listed for select, I am not sure what to do for favorite folders.

Do you have any suggestion?

**Comment by carlfriedrich** [comment]
@leftstick Thanks for the reply. I don't see why folders should be a problem, you could just display them along with the files, and if a folder is selected, display the folder's contents. Just like when you open a file from the file system. Wouldn't that work?

![2](https://user-images.githubusercontent.com/45259958/150785866-3262f908-743a-44b3-9296-ef6444987590.gif)

**Comment by leftstick** [comment]
@carlfriedrich you are right. It should not be a problem. My concern is about user experience. I will think about the default. Or PR is very welcomed
